Description: Fix parallel build
 libharvid is required to build the harvid binary in the src directory.
 Declaring this dependency is only possible, when changing the $(SUBDIRS)
 target from double-colon to single-colon, since double-colon implies
 independence.
 But then it has to be marked .PHONY as well, or nothing will be built.

Author: Andreas Cadhalpun <Andreas.Cadhalpun@googlemail.com>
Last-Update: <2015-11-06>
Forwarded: robin@gareus.org

--- harvid-0.8.0.orig/Makefile
+++ harvid-0.8.0/Makefile
@@ -3,7 +3,9 @@ SUBDIRS = libharvid src doc
 
 default: all
 
-$(SUBDIRS)::
+src: libharvid
+
+$(SUBDIRS):
 	$(MAKE) -C $@ $(MAKECMDGOALS)
 
 all clean man install uninstall install-bin install-man uninstall-bin uninstall-man install-lib uninstall-lib: $(SUBDIRS)
@@ -11,4 +13,4 @@ all clean man install uninstall install-bin install-man uninstall-bin uninstall-
 dist:
 	git archive --format=tar --prefix=harvid-$(VERSION)/ HEAD | gzip -9 > harvid-$(VERSION).tar.gz
 
-.PHONY: clean all subdirs install uninstall dist install-bin install-man uninstall-bin uninstall-man install-lib uninstall-lib
+.PHONY: clean all subdirs install uninstall dist install-bin install-man uninstall-bin uninstall-man install-lib uninstall-lib $(SUBDIRS)

